### PR TITLE
Issue51 @family

### PR DESCRIPTION
**Pull request by RCinbo**
'has_concept("keyword")' toegevoegd in de yaml
@family keyword toegevoegd in ieder van de R scripts in de package

### CODE REVIEW

File: R/read_kmi_data.R [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #' @importFrom rlang .data
 #' @importFrom lubridate tz
 #'
+#' @family download functions
```
Comment (hansvancalster) [inline R/read_kmi_data.R]:
```suggestion
#' @family download_functions
```

File: R/read_kml_file.R [hunk 2]
```diff
@@ -9,6 +9,8 @@
 #' @importFrom rlang .data
 #' @importFrom lubridate dmy_hm
 #'
+#' @family download functions
```
Comment (hansvancalster) [inline R/read_kml_file.R]:
```suggestion
#' @family download_functions
```

File: R/read_knmi_data.R [hunk 3]
```diff
@@ -17,6 +17,8 @@
 #' @importFrom lubridate ymd_h
 #' @importFrom rlang .data
 #'
+#' @family download functions
```
Comment (hansvancalster) [inline R/read_knmi_data.R]:
```suggestion
#' @family download_functions
```

File: R/read_mow_data.R [hunk 4]
```diff
@@ -17,6 +17,8 @@
 #' @importFrom lubridate dmy_hms
 #' @importFrom rlang .data
 #'
+#' @family download functions
```
Comment (hansvancalster) [inline R/read_mow_data.R]:
```suggestion
#' @family download_functions
```

File: _pkgdown.yml [hunk 5]
```diff
@@ -8,46 +8,35 @@ reference:
...
     desc: "Functions to download/read data or specific data formats"
     contents:
-      - read_mow_data
-      - download_knmi_data_hour
-      - download_zenodo
-      - read_knmi_data
-      - read_kmi_data
-      - read_kml_file
-      - extract_soil_map_data
       - has_concept("download")
```
Comment (hansvancalster) [inline _pkgdown.yml]:
```suggestion
      - has_concept("download_functions")
```

**Review by hansvancalster — CHANGES_REQUESTED** [review]
Great! Looks good. If you accept my suggestions online, can you also rerun `devtools::document` after pulling the remote changes?

**Review by hansvancalster — APPROVED** [review]

**Comment by hansvancalster** [comment]
closes #51

**Comment by hansvancalster** [comment]
closes #34

**Comment by hansvancalster** [comment]
@damianooldoni @niconoe can you have a look at the failing checks? Looks like they have to do with failing unit tests for `gbif_species_name_match` and `setup_coding_club_session`

**Comment by damianooldoni** [comment]
Thanks @hansvancalster, it is on my to do list of tomorrow!

**Comment by damianooldoni** [comment]
@hansvancalster: I removed the section with the three failing tests as they are not really important.

**Pull request merged**